test/Dapr.Client.Test/StateApiTest.cs

```diff
@@ -224,6 +224,135 @@ public async Task SetStateAsync_ThrowsForNonSuccess()
             await FluentActions.Awaiting(async () => await task).Should().ThrowAsync<RpcException>();
         }
 
+        [Fact]
+        public async Task ExecuteStateTransactionAsync_CanSaveState()
+        {
+            // Configure Client
+            var httpClient = new TestHttpClient();
+            var daprClient = new DaprClientBuilder()
+                .UseGrpcChannelOptions(new GrpcChannelOptions { HttpClient = httpClient })
+                .Build();
+
+            var stateValue1 = new Widget() { Size = "small", Color = "yellow", };
+            var metadata1 = new Dictionary<string, string>()
+            {
+                {"a", "b" }
+            };
+            var options1 = new StateOptions();
+            options1.Concurrency = ConcurrencyMode.LastWrite;
+            ;
```

Comment:
`;` 

Is probably the most abstract line of code I've seen in a while.

Reply:
:)